CribbageTournamentTracker/src/routes/+page.svelte

```diff
@@ -18,7 +18,18 @@
       </div>
     </div>
     <div class="col-md-9 col-sm-12 bg-dark">
-      Main Content
+      <div class="text-center">
+        <button class="btn btn-primary" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasScrolling" aria-controls="offcanvasScrolling">Team Chat</button>
+      </div>
+    </div>
+    <div class="offcanvas offcanvas-end" data-bs-scroll="true" data-bs-backdrop="false" tabindex="-1" id="offcanvasScrolling" aria-labelledby="offcanvasScrollingLabel" style="margin-top: 3.6rem">
```

Comment:
nit: looked into bootstrap styles and looks like they don't have our number for this (56 px or 4rem). For now, could we replace `style="margin-top: 3.6rem"` with a custom CSS class?